polygon/bridge/reader.go

```diff
@@ -46,25 +47,22 @@ func NewReader(store Store, log log.Logger, stateReceiverContractAddress string)
 func (r *Reader) Events(ctx context.Context, blockNum uint64) ([]*types.Message, error) {
 	start, end, err := r.store.EventIDRange(ctx, blockNum)
 	if err != nil {
-		return nil, err
-	}
-
-	if end == 0 { // exception for tip processing
```

Comment:
Why are you no longer checking this?

Reply:
Because end can no longer be 0. `ErrEventIDRangeNotFound` is returned if the map is not found.